rules/crowdstrike_rules/crowdstrike_detection_passthrough.yml

```diff
@@ -6,6 +6,7 @@ Enabled: true
 Severity: Medium
 LogTypes:
   - Crowdstrike.DetectionSummary
+  - Crowdstrike.FDREvent
```

Comment:
That is a good question.

> in terms of logical operands and ingestion, what is the rule here?
> Would we expect each detection which has Crowdstrike.FDREvent added to it to create duplicate alerts due to the differing logtypes?

It is an `OR`, in the sense of that the event will either be an `Crowdstrike.FDREvent` or a `Crowdstrike.DetectionSummary`, when coming into panther. And yes, we need one rule for each logtypes during the migration.
To clarify, we are in the midst of a migration from `Crowdstrike.*` to `Crowdstrike.FDREvent`. We will also have a period, for which we will have "dual writes" enabled (meaning both `Crowdstrike.FDREvent` and `Crowdstrike.DetectionSummary` could be written), but that is to be expected.

> perhaps because the detection shares a func dedup(), we would only expect one alert per dedup.

I'm not sure it will work as the deduplication works on the basis of the type too. I'll need to check.

Reply:
[We think there will be one alert after all](https://github.com/panther-labs/panther-enterprise/blob/83763db8a0acce956f34d6028e9519473d5dff25/internal/core/detections_engine/src/alert_merger.py#L236)